### PR TITLE
Fix deprecation warning coming for chef client 17: unified_mode true

### PR DESCRIPTION
**Pull request by RajeshPaul38**
Signed-off-by: Rajesh Paul <rajesh.paul@progress.com>

Added unified_mode true

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 version '1.0.1'
 source_url 'https://github.com/computology/packagecloud-cookbook' if respond_to?(:source_url)
```
Comment (tas50) [inline metadata.rb:8]:
```suggestion
source_url 'https://github.com/computology/packagecloud-cookbook'
```

File: metadata.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
 issues_url 'https://github.com/computology/packagecloud-cookbook/issues' if respond_to?(:issues_url)
```
Comment (tas50) [inline metadata.rb:9]:
```suggestion
issues_url 'https://github.com/computology/packagecloud-cookbook/issues'
```

**Comment by RajeshPaul38** [comment]
applied your suggestions @tas50

**Comment by damacus** [comment]
Resolved via #66

**Pull request closed without merging**